MAINTAINERS

```diff
@@ -124,8 +124,93 @@ the relevant operators.
 
 * If the change affects the governance, philosophy, goals or principles of the project,
 it must be approved by BDFL.
+
+* A pull request can be in 1 of 5 distinct states, for each of which there is a corresponding label
+that needs to be applied. `Rules.review.states` contains the list of states with possible targets
+for each.
 """
 
+		# Triage
```

Comment:
I assume a bot will assign new PRs this label, right?  If not, we could say "no label" == "triage" implicitly.